docs/writing-tech-docs/getting-started.md

```diff
@@ -0,0 +1,53 @@
+# Documentation Quickstart Guide 
+
+Want to publish documentation? This quick start guide will provide you with the necessary steps to get you from API Specification to ready-to-publish API Documentation in 5 minutes flat. 
+
+## 1. Open [Stoplight Studio](https://stoplight.io/p/studio)
```

Comment:
Can we put links in content instead of in headers? this is not something I see outside of stoplight.

Reply:
sure, not a problem